EIPS/eip-epbs.md

```diff
@@ -0,0 +1,260 @@
+
+This asynchronous mechanism has some consequences as slots may be *empty* as defined above. In these cases, the consensus layer does not process any more withdrawals until an execution payload has fulfilled the outstanding ones. An alternative design would be to defer all of withdrawal processing to the execution payload validation phase (ie. `process_execution_payload`). This has the advantage of not needing to track the fulfilled withdrawals on the beacon chain. The logic changes when several payloads are missing, in which case balances on the beacon chain change and therefore a withdrawal that would be possible with the former mechanism may be different, or even impossible with the latter.
+
+### Three state transition functions
+
+The current EIP adds an extra state transition function to the block processing in Ethereum. Processing a `SignedBeaconBlock` changes the consensus layer `BeaconState`. A `SignedExecutionPayloadEnvelope` changes both the execution layer state and the consensus layer one. As such, the envelope commits to the consensus layer post-state-transition beacon state root. 
+
+### Compatible designs 
+
+#### Inclusion lists
+
+This EIP is fully compatible with forward inclusion lists as specified in [EIP-7547](./eip-7547.md) or similar.
+
+#### Slot auctions
+
+A simple change to this EIP is to remove the blockhash commitment from the `SignedExecutionPayloadHeader`. This allows the builder to commit any payload to the slot. A preliminary security analysis shows that payload equivocation does not weaken forkchoice's FFG. Some advantages of Slot auctions include:
+
+- Better user experience as any submitted transaction can be included in the next block (with block auctions a transaction sent in the first half of the slot can only be included in the following block). 
+- Longer continuous time to build blocks. 
+- Better compatibility designs with forkchoice enforced inclusion list proposals. 
+
+
+## Backwards Compatibility
+
+This EIP introduces backward incompatible changes to the block validation rule set on the consensus layer and must be accompanied by a hard fork. 
+
+## Security Considerations
+
+### Reorg resistance 
+
+`PROPOSER_SCORE_BOOST` is reduced from 40 to 20. This does not allow however a en-anti reorg by a proposer with 20% of the total stake as the attacker's payload will also be not included. Proposers are protected against *1-slot sandwich* reorgs by a colluding set of validators and builders controlling more than 20% of the total stake
+
+### Builder safety
+
+- A colluding set of proposers and attesters controlling consecutive blocks and more than 20% of the total stake can reorg a builder's payload and force it to pay the bid's value. 
+- There is no possible *unbundling* of the builder's payload, that is, if the builder reveals a payload for the head of the chain, no other payload is possible for the current slot. 
```

Comment:
"No possible unbundling" it sounds like you are talking about equivocation? Unbundling = other parties can re-include stuff from the builder's block in the canonical chain, forking the builder's payload out

Reply:
Yes this is impossible as l well as long as the builder locks txs to the slot. On MEV boost you can actually unbundle in the same slot, on ePBS this is impossible

Reply:
added the words to make " There is no possible *unbundling* of the builder's payload in the same slot,..."

Reply:
> Yes this is impossible as l well as long as the builder locks txs to the slot. On MEV boost you can actually unbundle in the same slot, on ePBS this is impossible

I am not sure the builder can lock txs to the slot, but the tx senders can do that no?